eu.nokun.MirrorHall.json

```diff
@@ -51,10 +51,42 @@
         "python3-requirements.json",
         {
             "name": "inetutils",
-            "buildsystem": "simple",
-            "build-commands": [
-                "./configure --prefix=/app --libexec=/app/bin --localstatedir=/var --sysconfdir=/etc --without-{wrap,pam} --disable-{dnsdomainname,ftp,ftpd,ifconfig,inetd,logger,ping,ping6} --disable-{rcp,rexec,rexecd,rlogin,rlogind,rsh,rshd,syslogd} --disable-{talk,talkd,telnet,telnetd,tftp,tftpd,traceroute,uucpd,whois} --enable-hostname",
-                "make -j ${FLATPAK_BUILDER_N_JOBS}",
+            "buildsystem": "autotools",
+            "config-opts": [
+                "--prefix=/app",
+                "--libexec=/app/bin",
+                "--localstatedir=/var",
+                "--sysconfdir=/etc",
+                "--without-wrap",
+                "--without-pam",
+                "--disable-dnsdomainname",
+                "--disable-ftp",
+                "--disable-ftpd",
+                "--disable-ifconfig",
+                "--disable-inetd",
+                "--disable-logger",
+                "--disable-ping",
+                "--disable-ping6",
+                "--disable-rcp",
+                "--disable-rexec",
+                "--disable-rexecd",
+                "--disable-rlogin",
+                "--disable-rlogind",
+                "--disable-rsh",
+                "--disable-rshd",
+                "--disable-syslogd",
+                "--disable-talk",
+                "--disable-talkd",
+                "--disable-telnet",
+                "--disable-telnetd",
+                "--disable-tftp",
+                "--disable-tftpd",
+                "--disable-traceroute",
+                "--disable-uucpd",
+                "--disable-whois",
+                "--enable-hostname"
+            ],
+            "post-install": [
                 "install -Dm755 ./src/hostname /app/bin/hostname"
```

Comment:
why do you need this?

```
[📦 org.freedesktop.Sdk ~]$ which hostname
/usr/bin/hostname
```

Reply:
That was due to a mDNS issue in the past, but should not be needed anymore. I removed it for now.